scipy/signal/_ltisys.py

```diff
@@ -3436,9 +3436,9 @@ def dbode(system, w=None, n=100):
             * 4 (A, B, C, D, dt)
 
     w : array_like, optional
-        Array of frequencies (in radians/sample). Magnitude and phase data is
-        calculated for every value in this array. If not given a reasonable
-        set will be calculated.
+        Array of frequencies (in radians/sample, i.e. 0 to pi covers up to the
+        Nyquist frequency). Magnitude and phase data is calculated for every
+        value in this array. If not given, a reasonable set will be calculated.
```

Comment:
Sorry for the late response. In my understanding the correct term would [angular frequency]. I would propose something like
```text
Array of angular frequencies (in `2*pi/sampling interval`)  ...
```


[angular frequency]: https://en.wikipedia.org/wiki/Angular_frequency

Reply:
I don't think "angular frequency" is correct. `radians/time` is angular frequency, and it is the frequency, in `Hz`, for example, multiplied by a factor of `2*pi`.

`radians/sample`, as we are using them here, are a mapping from 0 Hz to the nyquist frequency (pi) and to the sampling rate (2 pi), and are specifically relevant for discrete systems.

Reply:
You are right, it is not an angular frequency, but a  [normalized (angular) frequency] (should have read my own reference more carefully :sweat_smile:). If _angular frequency_ is not correct, _frequency_ isn't the correct term either:

* For a _frequency_ $f_x$ and sampling interval $T$ , one writes: $x[k] = a_x cos(2\pi f_x k T)$
* For a _angular_ frequency $\omega_x$ and sampling interval $T$:  $x[k] = a_x  cos(\omega_x k T)$
* For a  [normalized (angular) frequency] $\tilde{\omega_x}$, one writes: $x[k] = a_x\cos(\tilde{\omega_x} k)$.

I would prefer a  formulation along the line of  either 
```
Array of normalized frequencies (...
``` 
or 
```
Array of frequencies normalized to the Nyquist frequency being π with unit `radiant / sample)...
```



[normalized (angular) frequency]: https://en.wikipedia.org/wiki/Normalized_frequency_(signal_processing)

Reply:
I like that idea! Will update the PR